degu_gw_manual/00_intro.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: default
+title: Deguゲートウェイマニュアル
+nav_order: 3
+has_children: true
+permalink: technical_specifications
```

Comment:
This link may  "degu_gw_manual". I fix it in following patch.
https://github.com/open-degu/open-degu.github.io/commit/5cbb16cc750186f1f0502fac170f0be4806223fe

Thank you.